code/__HELPERS/files.dm

```diff
@@ -19,14 +19,24 @@ GLOBAL_VAR_INIT(fileaccess_timer, 0)
 		var/list/choices = flist(path)
 		if(path != root)
 			choices.Insert(1,"/")
+		var/list/sorted_choices = sort_list(choices)
+		choices.Insert(length(choices) + 1, "Download Folder")
```

Comment:
You're not using `choices` anymore, so this is dead code.

Replace with this:

```suggestion
		choices = sort_list(choices) + "Download Folder"
```

...then use `choices` instead of `sorted_choices`.

Reply:
whoops, fixed